cpp/tests/reductions/reduction_tests.cu

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <iterator>
+#include <type_traits>
+#include <stdlib.h>
+
+#include "gtest/gtest.h"
+
+#include <cudf.h>
+#include <cudf/functions.h>
+
+#include <thrust/device_vector.h>
+
+#include "tests/utilities/cudf_test_fixtures.h"
+#include "tests/utilities/cudf_test_utils.cuh"
+#include "tests/utilities/column_wrapper.cuh"
+
+template <typename T>
+T getValueFromScalar(gdf_scalar& scalar)
```

Comment:
I think this is unnecessary.

Reply:
I will use `scalar_wrapper` in `utilities_tests` instead. (scalar_wrapper was not exist when I wrote `getValueFromScalar`).